test/lock.test.ts

```diff
@@ -44,4 +48,52 @@ describe("Lock", () => {
     expect(sharedResource).toContain("b");
     expect(sharedResource).toEqual(r);
   });
+
+  test("Lock store behaves correctly when acquiring and releasing locks", async () => {
+    const acquireResult = await lockStore.tryAcquire("resource-id-1", "execution-id-1");
+    expect(acquireResult).toBe(true);
+
+    if (useDurable) {
```

Comment:
If the local store is behaving differently than the remote then we need to change one of them to match the other